CHANGELOG.md

```diff
@@ -1,3 +1,97 @@
+### citus v10.1.0 (June 15, 2021) ###
+
+* Drops Postgres 11 support
+
+* Fixes a bug that can cause a crash when DEBUG4 logging is enabled
+
+* Fixes column default coming from a sequence
+
+* Fixes data race in `get_rebalance_progress`
+
+* Fixes error message for local table joins
+
+* Fixes FROM ONLY queries on partitioned tables
+
+* Fixes issues caused by public schema being omitted in queries
+
+* Fixes matview access method change issue
+
+* Fixes nested select query with union bug
+
+* Fixes possible segfaults when using Citus in the middle of an upgrade
+
+* Fixes problems with concurrent calls of `DropMarkedShards`
+
+* Fixes relname null bug when parallel execution
+
+* Fixes shared dependencies that are not resident in a database
+
+* Fixes stale hostnames bug in prep. statements after `master_update_node`
```

Comment:
I think we can extend prep. -> prepared. We can make use of an extra line if this exceeds the line.